sherpa/fit.py

```diff
@@ -572,18 +646,40 @@ def primini(self, statfunc, pars, parmins, parmaxes, statargs=(),
 
         This is a chi-square statistic where the variance is computed
         from model amplitudes derived in the previous iteration of the
-        fit. This 'Iterative Weighting' ([1]_) attempts to remove
+        fit. This 'Iterative Weighting' [1]_ attempts to remove
         biased estimates of model parameters which is inherent in
-        chi-square2 statistics ([2]_).
+        chi-square statistics [2]_.
+
+        Raises
+        ------
+        sherpa.utils.err.FitErr
+            This exception is raised if the statistic is not
+            supported. This method can only be used with
+            Chi-Square statistics with errors.
+
+        Notes
+        -----
+        The following keys are looked for in the ``itermethod_opts``
+        dictionary:
+
+        ========  =========  ===========
+        Key       Type       Description
+        ========  =========  ===========
+        tol       number     The tolerance used when comparing the statistic
+                             values.
+        maxiters  int        The maximum number of iterations.
+        ========  =========  ===========
 
         The variance in bin i is estimated to be:
 
-        sigma^2_i^j = S(i, t_s^(j-1)) + (A_s/A_b)^2 B_off(i, t_b^(j-1))
+        .. math::
+
+           \sigma(j)_i^2 = S(i, t_s(j-1)) + (A_s/A_b)^2 B_{off}(i, t_b(j-1))
```

Comment:
This output be seen at http://hea-www.harvard.edu/~dburke/playground/sherpa/fit/api/sherpa.fit.IterFit.html#sherpa.fit.IterFit.primini - so it is being rendered via a LaTeX-like system. There is a discussion to be had over how much extra markup we want to include in the docstrings to make the HTML output nicer while making the docstrings harder to read.

There's actually two reasons for changing this equation:

a) making it latex-able
b) changing the syntax - it appears that t_a^b was being used to say "t_a" at iteration b, and I've changed this to "t_a(b)" which I think is a bit less confusing, but it'll be interesting to see what @anetasie thinks